### PR TITLE
update nix in dockerfile to 1.10

### PR DESCRIPTION
**Pull request by offlinehacker**

### CODE REVIEW

File: misc/docker/Dockerfile [hunk 1]
```diff
@@ -1,23 +1,22 @@
 FROM busybox
 
 RUN set -x \
-    && wget -O- http://nixos.org/releases/nix/nix-1.9/nix-1.9-x86_64-linux.tar.bz2 | \
+    && wget -O- http://nixos.org/releases/nix/nix-1.10/nix-1.10-x86_64-linux.tar.bz2 | \
```
Comment (edolstra) [inline misc/docker/Dockerfile]: Maybe we can generate this file, substituting the current version?

Comment (offlinehacker) [inline misc/docker/Dockerfile]: With a bash script?

Comment (edolstra) [inline misc/docker/Dockerfile]: No, there is a generic make rule for doing substitutions. See e.g. `misc/systemd/local.mk` and `misc/systemd/nix-daemon.service.in`.

Comment (offlinehacker) [inline misc/docker/Dockerfile]: Well in future i want to make automated builds on docker hub, so nix image will be allways up-to-date in that case i can't simply make substitutions.

File: misc/docker/Dockerfile [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
         bzcat - | tar xf - \
     && echo "nixbld:x:30000:nixbld1,nixbld10,nixbld2,nixbld3,nixbld4,nixbld5,nixbld6,nixbld7,nixbld8,nixbld9" >> /etc/group \
     && for i in $(seq 1 9); do echo "nixbld$i:x:3000$i:30000:::" >> /etc/passwd; done \
```
Comment (domenkozar) [inline misc/docker/Dockerfile]: doesn't this need to be `$(seq 1 10)` given that `nixbld10` exists in `/etc/group`

Comment (offlinehacker) [inline misc/docker/Dockerfile]: Yes, and also `3000$i` has to be changed to `$((30000 + $i))`

**Comment by domenkozar** [comment]
@offlinehacker ping

**Comment by domenkozar** [comment]
@offlinehacker time for 1.11 :)

**Comment by domenkozar** [comment]
I've made necessary changes in https://github.com/NixOS/nix/pull/852, this can be closed now.

**Comment by domenkozar** [comment]
@offlinehacker can you close this?

**Comment by offlinehacker** [comment]
Thanks, yes we can close now, the only thing left is to setup automated build on dockerhub.

**Pull request closed without merging**